sambacc/ctdb.py

```diff
@@ -572,7 +573,23 @@ def monitor_cluster_meta_changes(
         if nodes_file_path:
             _logger.info("updating nodes file: %s", nodes_file_path)
             _save_nodes(nodes_file_path, expected_nodes)
-        _maybe_reload_nodes(leader_locator, reload_all=reload_all)
+        _maybe_reload_nodes_retry(leader_locator, reload_all=reload_all)
+
+
+def _maybe_reload_nodes_retry(
+    leader_locator: typing.Optional[leader.LeaderLocator] = None,
+    reload_all: bool = False,
+    *,
+    tries: int = 5,
+) -> None:
+    for idx in range(tries):
+        time.sleep(1 << idx)
```

Comment:
Minor: have upper bound to sleep time. Something like `time.sleep(min(1 << idx, 60))`. Just in case someone naively increases `tries` to large value.